internal/provider/oci/images.go

```diff
@@ -14,7 +14,6 @@ import (
 	"github.com/juju/errors"
 	ociCore "github.com/oracle/oci-go-sdk/v65/core"
 
-	"github.com/juju/juju/core/arch"
 	corearch "github.com/juju/juju/core/arch"
```

Comment:
Again here, the non-aliased one is removed. This contradicts file `internal/provider/oci/images_integration_test.go`.

Reply:
Consistency across packages, I think, is probably a utopia. In this case, I picked `corearch` since, again, there were some awkward name clashes